docs/nx-cloud/enterprise/dte/bitbucket-dte.md

```diff
@@ -0,0 +1,51 @@
+# Custom Distributed Task Execution on Bitbucket Pipelines
+
+Using [Nx Agents](/ci/features/distribute-task-execution) is the easiest way to distribute task execution, but it your organization may not be able to use hosted Nx Agents. With an [enterprise license](https://nx.app/enterprise), you can set up distributed task execution on your own CI provider using the recipe below.
+
+## Run Custom Agents on Bitbucket Pipelines
+
+Run agents directly on Bitbucket Pipelines with the workflow below:
+
+```yaml {% fileName="bitbucket-pipelines.yml" %}
+image: node:20
+
+clone:
+  depth: full
+
+definitions:
+  steps:
+    - step: &agent
+        name: Agent
+        script:
+          - export NX_BRANCH=$BITBUCKET_PR_ID
```

Comment:
Same as above: We are missing the `NX_CLOUD_DISTRIBUTED_EXECUTION_AGENT_COUNT: 3` env variable